arm-monitor/2017-04-01/swagger/actionGroups_API.json

```diff
@@ -0,0 +1,506 @@
+                        "$ref": "../examples/createOrUpdateActionGroup.json"
+                    }
+                },
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ResourceGroupNameParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ActionGroupNameParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    },
+                    {
+                        "name": "actionGroup",
+                        "in": "body",
+                        "required": true,
+                        "schema": {
+                            "$ref": "#/definitions/ActionGroupResource"
+                        },
+                        "description": "The action group to create or use for the update."
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "An existing action group was successfully updated.",
+                        "schema": {
+                            "$ref": "#/definitions/ActionGroupResource"
+                        }
+                    },
+                    "201": {
+                        "description": "A new action group was successfully created.",
+                        "schema": {
+                            "$ref": "#/definitions/ActionGroupResource"
+                        }
+                    },
+                    "default": {
+                        "description": "An error occurred and the action group could not be created or updated."
+                    }
+                }
+            },
+            "get": {
+                "description": "Get an action group.",
+                "operationId": "ActionGroups_Get",
+                "x-ms-examples": {
+                    "Get an action group": {
+                        "$ref": "../examples/getActionGroup.json"
+                    }
+                },
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ResourceGroupNameParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ActionGroupNameParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "The request succeeded.",
+                        "schema": {
+                            "$ref": "#/definitions/ActionGroupResource"
```

Comment:
The patch operation only supports tags, so this was intentionally omitted according to the [swagger checklist](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md):

> M2056: If the service (in Azure) only supports PATCHING tags on a Resource in the PATCH operation and has a PUT on the same resource which is named as "CreateOrUpdate", then it MUST NOT expose the "PATCH" operation in the swagger spec.